src/core/tests/type_prop/scatter_nd_update.cpp

```diff
@@ -109,84 +122,73 @@ TEST(type_prop, scatter_nd_update_fail_indices_last_dim) {
     }
 }
 
-using namespace ov::opset10;
-using namespace testing;
-
-class TypePropScatterUpdateNDV3Test : public TypePropOpTest<op::v3::ScatterNDUpdate> {
-protected:
-    void SetUp() override {
-        set_shape_symbols(data_3d_dynamic);
-    }
-    PartialShape data_3d_dynamic{{2, 5}, 2, {4, 10}};
-};
-
-TEST_F(TypePropScatterUpdateNDV3Test, data_input_partial_shape_and_symbols_propagation) {
-    const auto d = std::make_shared<Parameter>(element::f32, data_3d_dynamic);
+TYPED_TEST_P(TypePropScatterNDUpdateTest, data_input_partial_shape_and_symbols_propagation) {
```

Comment:
Add some dedicated test for new op to check new attribute is set correctly, has expected default value etc.

Reply:
Added 3 additional attribute tests for default, using constructor and setter